Formula/uhdm.rb

```diff
@@ -0,0 +1,54 @@
+class Uhdm < Formula
+  include Language::Python::Virtualenv
+
+  desc "Universal Hardware Data Model, modeling of the SystemVerilog Object Model"
+  homepage "https://github.com/chipsalliance/UHDM"
+  url "https://github.com/chipsalliance/UHDM/archive/refs/tags/v1.71.tar.gz"
+  sha256 "3aa3783bc3fd5054bfe4347de1e2edfaa1bb3fbfbecdc3232413e826e95113c7"
+  license "Apache-2.0"
+  head "https://github.com/chipsalliance/UHDM.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.11" => :build
+  depends_on "six" => :build
+  depends_on "capnp"
+
+  resource "orderedmultidict" do
+    url "https://files.pythonhosted.org/packages/53/4e/3823a27d764bb8388711f4cb6f24e58453e92d6928f4163fdb01e3a3789f/orderedmultidict-1.0.1.tar.gz"
+    sha256 "04070bbb5e87291cc9bfa51df413677faf2141c73c61d2a5f7b26bea3cd882ad"
+  end
+
+  def install
+    venv = virtualenv_create(buildpath/"venv", "python3.11")
+    resources.each do |r|
+      venv.pip_install r
+    end
+    # Static Library
+    system "cmake", "-S", ".", "-B", "build_static", \
+      "-DBUILD_SHARED_LIBS=OFF", "-DUHDM_BUILD_TESTS=OFF", \
+      "-DUHDM_USE_HOST_GTEST=ON", "-DUHDM_USE_HOST_CAPNP=ON", \
+      "-DPython3_EXECUTABLE=#{buildpath}/venv/bin/python", *std_cmake_args
+    system "cmake", "--build", "build_static"
+    system "cmake", "--install", "build_static"
```

Comment:
Generally Homebrew software should not use static libraries

Reply:
It looks like this actually isn't the case, I see a large number of libraries shipping both shared and static. I think I'll do the same here if its all the same?

Reply:
140 out of ~6000 isn't that much and not all matches there are enabling the static library. As a policy, we are not enabling static libraries unless they are explicitly requested multiple times by users, which is not the case here.